samples/client/petstore/k6/script.js

```diff
@@ -23,182 +23,335 @@ let apiKey = "TODO_EDIT_THE_API_KEY";
+          
+          let params = {headers: {"Content-Type": "application/json", "Accept": "application/json"}};
+          
+          
+          let request = http.put(url, JSON.stringify(body), params);
+          
+          
+          
+           sleep(SLEEP_DURATION);
+        }
+        
+        
         // Request No. 2
-        // TODO: edit the parameters of the request body.
-        body = {"id": "long", "category": {"id": "long", "name": "string"}, "name": "string", "photoUrls": "list", "tags": [{"id": "long", "name": "string"}], "status": "string"};
-        params = {headers: {"Content-Type": "application/json"}};
-        request = http.post(url, body, params);
-        sleep(SLEEP_DURATION);
+        {
+          let url = BASE_URL + `/pet`;
+          // TODO: edit the parameters of the request body.
+          let body = {"id": "long", "category": {"id": "long", "name": "string"}, "name": "string", "photoUrls": "list", "tags": "list", "status": "string"};
+          
+          let params = {headers: {"Content-Type": "application/json", "Accept": "application/json"}};
+          
+          
+          let request = http.post(url, JSON.stringify(body), params);
+          
+          
+          
+        }
+        
     });
-    group("/pet/findByStatus", () => {
-        let status = 'TODO_EDIT_THE_STATUS'; // specify value as there is no example value for this parameter in OpenAPI spec
-        let url = BASE_URL + `/pet/findByStatus?status=${status}`;
+    group("/user/{username}", () => {
+        let username = 'TODO_EDIT_THE_USERNAME'; // specify value as there is no example value for this parameter in OpenAPI spec
+        
         // Request No. 1
-        let request = http.get(url);
-        check(request, {
+        {
+          let url = BASE_URL + `/user/${username}`;
+          
+          
+          
+          let request = http.get(url);
+          
+          check(request, {
             "successful operation": (r) => r.status === 200
-        });
-        sleep(SLEEP_DURATION);
+          });
+          
+          
+           sleep(SLEEP_DURATION);
+        }
+        
+        
+        // Request No. 2
+        {
+          let url = BASE_URL + `/user/${username}`;
+          
+          
+          let request = http.del(url);
+          
+          
+          
+          
+        }
+        
     });
-    group("/pet/findByTags", () => {
-        let tags = 'TODO_EDIT_THE_TAGS'; // specify value as there is no example value for this parameter in OpenAPI spec
-        let url = BASE_URL + `/pet/findByTags?tags=${tags}`;
+    group("/pet/findByStatus", () => {
+        let status = 'TODO_EDIT_THE_STATUS'; // specify value as there is no example value for this parameter in OpenAPI spec
+        
         // Request No. 1
-        let request = http.get(url);
-        check(request, {
+        {
+          let url = BASE_URL + `/pet/findByStatus?status=${status}`;
+          
```

Comment:
still triple empty lines :P

Reply:
I think I haven't regenerated the file.

Reply:
Addressed.

Reply:
Interesting, the pipeline passed on `5082527` when the sample project was out of date.

Reply:
That was strange to me, too.